symbolic-minidump/src/cfi.rs

```diff
@@ -41,8 +41,22 @@ use symbolic_debuginfo::pdb::PdbObject;
 use symbolic_debuginfo::pe::{PeObject, RuntimeFunction, UnwindOperation};
 use symbolic_debuginfo::{Object, ObjectError, ObjectLike};
 
+/// The magic file preamble to identify cficache files.
+/// Files with version < 2 do not have a magic file preamble.
+pub const CFICACHE_MAGIC: [u8; 4] = *b"CFIC";
+
 /// The latest version of the file format.
-pub const CFICACHE_LATEST_VERSION: u32 = 1;
+pub const CFICACHE_LATEST_VERSION: u32 = 2;
+
+// The preamble are 8 bytes, all ASCII.
+// This should be forward-compatible with a binary format:
+// `CFICxxx\n`
+//      ^^^ <- 3 ASCII digits denoting the version, `002` for now.
```

Comment:
Can we make this binary right away, just like the symcache? This time around, it would also be great to fix the byte order rather than relying on native endianness.